src/shared/links.component.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { navigateToUrl } from "single-spa";
+
+export const SlackLink = ({ children }) => (
+  <a
+    target="_blank"
```

Comment:
`rel="nooopener"` is generally a good practice for links to external websites. https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Reply:
Ok! I think there is a eslint that warns about this. Can I update the eslint-config with it? (eslintreact/jsx-no-target-blank)

Reply:
Sure!

Reply:
Sure, could you actually add it to https://github.com/joeldenning/eslint-config-react-important-stuff? That eslint config is used all over the place and so adding it to that config would be even better